util/aws/http_client.cc

```diff
@@ -230,7 +231,8 @@ io::Result<std::unique_ptr<FiberSocketBase>> HttpClient::Connect(const std::stri
   std::error_code ec = socket->Connect(ep);
   if (ec) {
     LOG(WARNING) << "aws: http client: failed to connect; host=" << host << "; error=" << ec;
-    socket->Close();
+    auto ec2 = socket->Close();
+    (void)ec2;
```

Comment:
You _could_ do:
```suggestion
    std::ignore = = socket->Close();
```
I'm not saying should _should_ though :rofl:

Reply:
(same for other places in this file)

Reply:
oh wow!